packages/ui/client/utils/escape.ts

```diff
@@ -0,0 +1,8 @@
+export function escapeHtml(unsafe: string) {
+  return unsafe
+    .replace(/&/g, '&amp;')
+    .replace(/</g, '&lt;')
+    .replace(/>/g, '&gt;')
+    .replace(/"/g, '&quot;')
+    .replace(/'/g, '&#39;')
+}
```

Comment:
It might be worth noting that the similar code is also used for junit xml.

https://github.com/vitest-dev/vitest/blob/d0c3a1f73df01789458ff60613443be0cf97f12d/packages/vitest/src/node/reporters/junit.ts#L54-L64

Is it preferred to move this into `@vitest/utils` and share everywhere?